Makefile

```
@@ -224,22 +314,46 @@
 	pnpm update
 
 down: ## Stop the docker containers
+ifeq ($(DIND), 1)
+	$(DOCKER_COMPOSE) $(COMPOSE_FILES) down --remove-orphans
+else
 	$(DOCKER_COMPOSE) down --remove-orphans
+endif
 
 sh: ## Log to the docker container
```

checkmake (0.2.2)

[warning] Target "sh" should be declared PHONY.

(phonydeclared)